www/main/shortcut_tray.js

```diff
@@ -2,12 +2,11 @@ const h = require('hyperscript');
 const { remote } = require('electron');
```

Comment:
See this error when Running the app 
![image](https://user-images.githubusercontent.com/2982784/104134204-10820900-533d-11eb-8134-ac81eb02f141.png)